libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -136,56 +137,21 @@ class DropInNavigationView @JvmOverloads constructor(
         attachStarted(
             MapCoordinator(navigationContext, binding.mapView),
             GuidanceCoordinator(navigationContext, binding.guidanceLayout),
-            InfoPanelCoordinator(navigationContext, binding.infoPanelLayout),
+            InfoPanelCoordinator(navigationContext, binding.infoPanelLayout, binding.guideBottom),
```

Comment:
Why do we need a `buideBottom`? The `infoPanelLayout` was supposed to encompass "anything at the bottom" for our portrait layout.

Reply:
My understanding is that `infoPanelLayout` is only BottomSheet.

So I moved `roadNameLayout` to  the `ConstraintLayout` and  anchored it to horizontal `Guideline` which position is updated by `BottomSheetBehavior.BottomSheetCallback` in InfoPanelCoordinator. I think this solution will not only allow us to position `roadNameLayout` above bottom sheet but will also allow us to freeze `roadNameLayout` position when bottom sheet is dragged to a fullscreen.

Reply:
ah was confused `guideBottom`. maybe `infoPanelGuideline`

Reply:
It's not confusing to me. I will rename it to `guidelineBottom`.